buildlib/azure-pipelines.yml

```diff
@@ -44,6 +44,7 @@ stages:
                  echo "##vso[task.logissue type=error]Bad commit title(s), see $url for more info."
                  echo "##vso[task.complete result=Failed;]"
               fi
+            condition: eq(variables['Build.Reason'], 'PullRequest')
```

Comment:
WDYT about checking System.PullRequest.TargetBranch is non-empty?

Reply:
`System.PullRequest` is not defined at all for non-PRs. IMO checking the `Build.Reason` is a reliable way to check what we need. Do you see any potential issues with that?

Reply:
ok